src/mavoscript.js

```diff
@@ -491,6 +491,14 @@ var _ = Mavo.Script = {
 			},
 			precedence: 2
 		},
+		"groupby": {
+			symbol: "by",
```

Comment:
I believe we don't need this line anymore since after the PR is merged, `a by b` will be the same as `group_by(a, b)`, and not `group(a, b)`. Do you mind trying to remove it and see what we get?